### PR TITLE
only use major version to specify python version

### PR DESCRIPTION
**Pull request by oToToT**
I think it is more flexible to use only major version of python. Also, miku seems only check the major version of python, so it might not be necessary to specify minor version in the code.

### CODE REVIEW

File: src/batchjudge.cpp [hunk 1]
```diff
@@ -52,9 +52,9 @@ int BatchJudge(int pid, int td, int boxid, int tl, int ml, int ol, int testee,
 
   // invoke box command
   if (lang == "python2") {
-    sandboxExec(boxid, opt, {"/usr/bin/env", "python2.7", "main.pyc"});
+    sandboxExec(boxid, opt, {"/usr/bin/env", "python2", "main.pyc"});
   } else if (lang == "python3") {
-    sandboxExec(boxid, opt, {"/usr/bin/env", "python3.7", "main.pyc"});
+    sandboxExec(boxid, opt, {"/usr/bin/env", "python3", "main.pyc"});
```
Comment (TaWeiTu) [inline src/batchjudge.cpp]: I think something like
```cpp
if (lang.startswith("python")) {
  sandboxExec(..., {"/usr/env/bin", lang, ...});
} else {
  // C++
}
```
is more elegant.

**Review by TaWeiTu — CHANGES_REQUESTED** [review]

File: src/batchjudge.cpp (hunk 1 above)
Comment (oToToT) [reply src/batchjudge.cpp]: How about something like this?  
use `lang.find_first_of("python") == 0` as `lang.starts_with`.

Maybe we could change this when C++20 being released officially.

Comment (TaWeiTu) [reply src/batchjudge.cpp]: LGTM

**Review by TaWeiTu — APPROVED** [review]
LGTM

**Pull request merged**